packages/php-wasm/logger/src/lib/logger.ts

```diff
@@ -307,34 +294,24 @@ export function collectPhpLogs(
  * @param worker The service worker
  */
 export function reportServiceWorkerMetrics(worker: ServiceWorkerGlobalScope) {
-	worker.addEventListener('activate', (event) => {
-		// Trigger controllerchange event to update the client count.
-		/**
-		 * Triggers the `controllerchange` event in other clients to update the count.
-		 *
-		 * > When a service worker is initially registered, pages won't use it until they next load.
-		 * > The claim() method causes those pages to be controlled immediately. Be aware that
-		 * > this results in your service worker controlling pages that loaded regularly over the network,
-		 * > or possibly via a different service worker.
-		 * 
-		 * @see https://developer.mozilla.org/en-US/docs/Web/API/Clients/claim
-		 */
-		event.waitUntil(worker.clients.claim());
-	});
-	worker.addEventListener('message', (event) => {
-		if (!event.source) {
-			return;
-		}
-		if (event.data === 'getServiceWorkerMetrics') {
-			worker.clients.matchAll().then((clients) => {
-				event.source!.postMessage({
-					numberOfOpenPlaygroundTabs: clients.filter(
-						// Only count top-level frames to get the number of tabs.
-						(c) => c.frameType === 'top-level'
-					).length,
+	worker.addEventListener('activate', () => {
+		worker.clients.matchAll().then((clients) => {
+			const metrics = {
+				numberOfOpenPlaygroundTabs: clients.filter(
+					// Only count top-level frames to get the number of tabs.
+					(c) => c.frameType === 'top-level'
+				).length,
+			};
+			worker.clients
+				.matchAll({
+					includeUncontrolled: true,
```

Comment:
Why includeUncontrolled? Let's leave a comment in the code to explain. Otherwise LGTM 👍

Reply:
@bgrgicak yeah I meant more not the literal meaning of that option but rather why bother with uncontrolled clients.

Reply:
I can update it in #1264 with something like this.

```
// Inform all Playground clients that there was a change in the number of active tabs
```

Reply:
Right, but why an uncontrolled client is considered a Playground client? It seems like that browser tab wouldn't be able to access Playground anyway, as being controlled is crucial for that, right?

Reply:
Correct, I tested this a bit and it's actually not needed, so I will remove this line in #1264